pages/docs/[section].vue

```diff
@@ -0,0 +1,113 @@
+<template>
```

Comment:
![image](https://user-images.githubusercontent.com/2015821/197932438-fb155c8a-d72e-457b-80a1-8a3a3c3d6951.png)

you get this in the article space if you change the slug to something that isn't a pre-existing article.

you also lose the document navigation section entirely so you're pretty much in a stuck state unless you click the docs button at the top again.

imo the list of all the docs on the left should stay there if the slug doesn't exist and also we should redirect somewhere.  one option is to the index page again.  another option is to some sorta 404 page, idk if we should do it for the site as a whole or just something within the docs construct.